Regression in event types

Issue caused by #7718 when attempting to build `@pixi/webworker` (no events).

```
node_modules/@pixi/display/lib/DisplayObject.d.ts:10:67 - error TS2694: Namespace 'GlobalMixins' has no exported member 'FederatedEventEmitterTypes'.

10 export declare type DisplayObjectEventEmitterTypes = GlobalMixins.FederatedEventEmitterTypes & {
```

Version: 7.0.0-beta.5

--- a/packages/display/global.d.ts
+++ b/packages/display/global.d.ts
@@ -11,4 +11,10 @@ declare namespace GlobalMixins
     {
 
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface DisplayObjectEvents
+    {
+
+    }
 }

--- a/packages/display/src/DisplayObject.ts
+++ b/packages/display/src/DisplayObject.ts
@@ -11,18 +11,19 @@ export interface IDestroyOptions
     baseTexture?: boolean;
 }
 
-export type DisplayObjectEventEmitterTypes = GlobalMixins.FederatedEventEmitterTypes & {
+export interface DisplayObjectEvents extends GlobalMixins.DisplayObjectEvents
+{
     added: [Container];
     childAdded: [DisplayObject, Container, number];
     childRemoved: [DisplayObject, Container, number];
     destroyed: [];
     removed: [Container];
     removedFrom: [DisplayObject, Container, number];
-};
+}
 
 export interface DisplayObject
-    extends Omit<GlobalMixins.DisplayObject, keyof utils.EventEmitter<DisplayObjectEventEmitterTypes>>,
-    utils.EventEmitter<DisplayObjectEventEmitterTypes> {}
+    extends Omit<GlobalMixins.DisplayObject, keyof utils.EventEmitter<DisplayObjectEvents>>,
+    utils.EventEmitter<DisplayObjectEvents> {}
 
 /**
  * The base class for all objects that are rendered on the screen.
@@ -209,7 +210,7 @@ export interface DisplayObject
  * one is also better in terms of performance.
  * @memberof PIXI
  */
-export abstract class DisplayObject extends utils.EventEmitter<DisplayObjectEventEmitterTypes>
+export abstract class DisplayObject extends utils.EventEmitter<DisplayObjectEvents>
 {
     abstract sortDirty: boolean;
 

--- a/packages/events/global.d.ts
+++ b/packages/events/global.d.ts
@@ -9,6 +9,12 @@ declare namespace GlobalMixins
 
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface DisplayObjectEvents extends FederatedEventEmitterTypes
+    {
+
+    }
+
     interface IRenderer
     {
         readonly events: import('@pixi/events').EventSystem;

